go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/giantswarm/micrologger v0.5.0
 	github.com/giantswarm/operatorkit/v4 v4.3.1
 	github.com/giantswarm/versionbundle v0.2.0
-	github.com/go-kit/log v0.1.0
+	github.com/go-kit/kit v0.10.0
```

Comment:
So this is needed after all?

Reply:
that's required by alertmanager/config: https://github.com/prometheus/alertmanager/blob/44f8adc06af5101ad64bd8b9c8b18273f2922051/config/coordinator.go#L21

Reply:
Right, got it.